app/components/online-button.js

```diff
@@ -0,0 +1,32 @@
+import Ember from "ember";
+
+export default Ember.Component.extend(Ember.TargetActionSupport, {
+  tagName: "div",
```

Comment:
No need to specify `div` by default components are div.

Reply:
sure, ill remove it.